MainModule/Server/Commands/Admins.lua

```diff
@@ -176,6 +176,7 @@ return function(Vargs, env)
 				if not target:find(":") then
```

Comment:
This could be changed to `if not target:find(":") then`.

Yes it's not part of the code that the pull changes but I think it could be changed